tools/tikv-control.md

```diff
@@ -179,3 +179,22 @@ If the command is successfully executed, it prints the above information. If the
     ```bash
     $ tikv-ctl --host 127.0.0.1:20160 region-properties -r 2
     ```
+
+### Modify the RocksDB configuration of TiKV dynamically
+
+The `modify-tikv-config` command is used to dynamically modify the configuration arguments. Currently it only supports dynamically modifying RocksDB related arguments. 
```

Comment:
- For the first sentence, this way sounds better and more user-centered: **You can use the** `modify-tikv-config` command **to** ...
- When we put "currently" at the beginning of a sentence, we usually add a comma after it and omit comma if it is in the middle.